catalog-info.yaml

```diff
@@ -143,3 +143,47 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: fleet-server-perf-tests
+  description: Pipeline tests for serverless scale testing a PR
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/fleet-server-perf-tests
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: fleet-server-perf-tests
+      description: fleet-server observability-perf tests
+    spec:
+      repository: elastic/fleet-server
+      pipeline_file: .buildkite/pipeline.perf-tests.yaml
+      branch_configuration: "main"
+      skip_intermediate_builds: false
+      provider_settings:
+        build_pull_request_forks: false
+        build_pull_requests: true
+        build_tags: false
+        build_branches: false
+        filter_enabled: true
+        filter_condition: >-
+          build.pull_request.id == null || (build.creator.name == 'elasticmachine' && build.pull_request.id != null)
+      cancel_intermediate_builds: true
+      cancel_intermediate_builds_branch_filter: '!main !7.* !8.*'
+      skip_intermediate_builds: true
+      skip_intermediate_builds_branch_filter: '!main !7.* !8.*'
+      teams:
+        ingest-fp:
+          access_level: MANAGE_BUILD_AND_READ
+        cloud-tooling:
+          access_level: MANAGE_BUILD_AND_READ
+        everyone:
+          access_level: READ_ONLY
```

Comment:
Would other users out of `ingest-fp` trigger this pipeline (commenting in a PR)? 

Currently, it looks good to just allow `ingest-fp`. Maybe at some point, it could be changed this setting for `everyone` to be BUILD_AND_READ?

Reply:
I'm not sure, i think we can decide on this later as well